<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>On successful push</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QCheckBox" name="saveLocalCopy">
             <property name="text">
              <string>Save local copy of image</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="playNotifcationSound">
             <property name="text">
              <string>Play notification sound</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="copyToClipboard">
             <property name="text">
              <string>Copy link to clipboad</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="pathLineEdit"/>
             </item>
             <item>
              <widget class="QToolButton" name="browseButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="openInBrowser">
             <property name="text">
              <string>Open link in browser</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="startHidden">
             <property name="text">
              <string>Start hidden</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Tray Icon Behaviour</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="showSettings">
             <property name="text">
              <string>Show settings dialog</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="beginScreenCapture">
             <property name="text">
              <string>Begin screen capture mode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="openUploadDialog">
             <property name="text">
              <string>Open upload file dialog</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Account</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Settings</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>API url :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="urlLineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>API key :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="apiLineEdit"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Key Bindings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Shortcuts</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="2">
            <widget class="QPushButton" name="fullscreenButton">
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Fullscren</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="ShortcutReader" name="shortcutFullscreen">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Ctrl+Shift+1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Current Window</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ShortcutReader" name="shortcutCurrentWindow">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Ctrl+Shift+2</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Area</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="ShortcutReader" name="shortcutArea">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Ctrl+Shift+3</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="areaButton">
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Upload file</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="ShortcutReader" name="shortcutUploadFile">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Ctrl+Shift+5</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="uploadFileButton">
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="currentWindowButton">
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="buttonQuit">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>427</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ShortcutReader</class>
   <extends>QLineEdit</extends>
   <header>shortcutreader.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
